<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreePrimeInput</class>
 <widget class="QWidget" name="ThreePrimeInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>25</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="pos3">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>0</y>
     <width>55</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G/C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="pos1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>55</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G/C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="pos2">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>0</y>
     <width>55</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G/C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
